ArmTemplates/azure-maps.md

```diff
@@ -8,6 +8,10 @@ azureMapsName: (required) string
 
 Name of Azure Maps. Will be created in the same resource group as the script is run and in the default location for resource group.
 
+location: (required) global
```

Comment:
The location is hardcoded so shouldn't be mentioned in this section.  Worth mentioning it on summary line under the title though